src/Home.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import useFetch from './hooks/useFetch'
+
+const Home = () =>{
+
+  const {data,loading} = useFetch(`https://source.unsplash.com/random/1920x1080`)
+
+  console.log(data)
```

Comment:
Avoid having console logs in your final app. It could give too much info to the end user in case they end up opening dev tools :)